modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -130,34 +105,37 @@ $ oc get pv
 $ oc create -n openshift-image-registry -f -
 ----
 +
+.Example PVC
 [source,yaml]
 ----
-apiVersion: "v1"
-kind: "PersistentVolumeClaim"
+apiVersion: v1
+kind: PersistentVolumeClaim
 metadata:
-  name: "image-registry-pvc"
+  name: image-registry-pvc
 spec:
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
       storage: 100Gi
-  storageClassName: nfs01
-  volumeMode: Filesystem
+  storageClassName: thin
 ----
+
+.. Verify that the PVC was created:
```

Comment:
In vSphere, OCP supports a default storage class which is the same as other cloudproviders, such as AWS, GCP, so there might be no different from the configuration of AWS EBS.
@wzheng1 Could you help double confirm? Thx!

Reply:
Yes, but image registry needs to create pvc manually. Since the default created pvc only supports RWX.

Reply:
I removed the entire optional step (`Optional: Add a new storage class to a PV`). Correct?